src/cortex_cli/cortex_cli.py

```diff
@@ -547,7 +547,10 @@ def logout(config_file: str, keep_tokens: str, force: bool) -> None:
             try:
                 logout_request(auth_server_url, realm, client_id, refresh_token)
             except (Timeout, ConnectionError, ClientAuthenticationError) as error:
-                raise click.ClickException(f'Error when logging out: {error}') from error
+                logger.warning(
+                    'Failed to revoke tokens due to error when connecting to authentication server: %s', error
+                )
+
             Process(pid).terminate()
             os.remove(tokens_file)
             logger.info('Logged out successfully.')
```

Comment:
I can imagine these two consecutive messages to be slightly confusing to some users:

```
Failed to revoke tokens due to error when connecting to authentication server...
Logged out successfully.
```

Perhaps, it can clarify the fact that even though tokens weren't revoked, the tokens file is deleted. Maybe, it's simply enough to change that last line to something like:


```python
logger.info('Tokens file deleted. Logged out.')
```

And omit `successfully` altogether :)

Reply:
Good suggestion, I have updated the messages